softlayer/resource_softlayer_scale_group.go

```diff
@@ -245,9 +245,8 @@ func buildScaleVlansFromResourceData(d *schema.Set, meta interface{}) ([]datatyp
 
 		if len(networkVlans) < 1 {
 			return nil, fmt.Errorf(
-				"Unable to locate a vlan matching the provided router hostname and vlan number: %s/%s",
-				primaryRouterHostname,
-				vlanNumber)
+				"Unable to locate the specified vlan: %d.%s",
+				vlanNumber, primaryRouterHostname)
```

Comment:
@riethm so what would be an improvement over `bcr01a.sjc03/1234` as it was before? Since that can't be any less ambiguous or confusing to the type of users you pose.